core/modules/replica/HttpProcessor.cc

```diff
@@ -453,7 +470,11 @@ void HttpProcessor::_initialize() {
         {"POST",   "/replication/v1/config/database", bind(&HttpProcessor::_addDatabaseConfig, self, _1, _2)},
         {"DELETE", "/replication/v1/config/table/:name", bind(&HttpProcessor::_deleteTableConfig, self, _1, _2)},
         {"POST",   "/replication/v1/config/table", bind(&HttpProcessor::_addTableConfig, self, _1, _2)},
-        {"POST",   "/replication/v1/sql/query", bind(&HttpProcessor::_sqlQuery, self, _1, _2)}
+        {"POST",   "/replication/v1/sql/query", bind(&HttpProcessor::_sqlQuery, self, _1, _2)},
```

Comment:
consider `lambda` rather than `bind` here?

Reply:
Tis is what I was trying to do at the beginning. Unfortunately, this makes the code to "swell" (not much, though). For example:
```
{"POST",   "/replication/v1/sql/query", bind(&HttpProcessor::_sqlQuery, self, _1, _2)}
```
Would turn into:
```
{"POST",   "/replication/v1/sql/query", [&self](qhttp::Request::Ptr const& req, qhttp::Response::Ptr const& resp) { self->_sqlQuery(req, resp); }}
```
I'm going to revisit this when I will be refactoring this code as proposed in:
https://confluence.lsstcorp.org/display/DM/Refactor+the+HTTP+Processor+of+the+Master+Replication+Controller